app/controllers/katello/api/v2/host_subscriptions_controller.rb

```diff
@@ -35,6 +37,42 @@ def events
       respond_for_index :collection => collection
     end
 
+    api :POST, "/hosts/subscriptions/register", N_("Simulate a host subscription-manager registration")
```

Comment:
If that is the case then I am losing track of why we need this special create when there is a host create already? Are we saying that each facet should provide an API method to create a host of that facet type?